apps/token-manager/contracts/TokenManager.sol

```diff
@@ -56,7 +56,6 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
     // We are mimicing an array in the inner mapping, we use a mapping instead to make app upgrade more graceful
     mapping (address => mapping (uint256 => TokenVesting)) internal vestings;
     mapping (address => uint256) public vestingsLengths;
-    mapping (address => bool) public everHeld;
```

Comment:
Can be safely removed because luckily it was the last storage mapping of the contract and it has never been used, although if we want to be really safe, we could leave a storage gap:

```
uint256 private everHeldDeprecationGap;
```